server/gnocco/server.go

```diff
@@ -11,6 +12,7 @@ import (
 	"github.com/miekg/dns"
 )
 
+// Server implements a cached DNS resolver
```

Comment:
I don't want to be nitpicking but if this is a Resolver, shouldn't it be called as such?
gnocco.Resolver sounds even better than gnocco.Server

Reply:
amended accordingly